src/sentry/templates/sentry/account/notifications.html

```diff
@@ -106,6 +106,23 @@
 
         <hr />
 
+        <h4>{% trans "Deploys" %}</h4>
+
+        <p>{% blocktrans %}Deploys are a summary of what releases are being pushed to what enviroment, and what commits are involved. Chose to recieve deploy emails for:{% endblocktrans %}</p>
```

Comment:
did you see the screen shot? it kind of makes sense in that context but I can make this a little less awkward